engine/plugins/fbs/response.js

```diff
@@ -193,12 +193,13 @@ Response.prototype.parseEncoding = function parseEncoding() {
             mappings.push(['onlineStatus', 1]);
             mappings.push(['checkIn', 1]);
             mappings.push(['checkOut', 1]);
+            mappings.push(['acsRenewalPolicy', 1]);
```

Comment:
Should be "renewalPolicy" to match the other ones?

Reply:
fixed